static/edx-pl/sass/main-ltr.scss

```diff
@@ -0,0 +1,22 @@
+// ------------------------------
```

Comment:
This is the SCSS of the pattern library, does this mean we expect to use the pipeline to compile it to CSS and then use that?